run.py

```diff
@@ -33,17 +37,80 @@ def _get_opt_parser(self):
         parser.add_option('-p', '--pass', dest='password', default=None)
         return parser
 
-    def _get_connection(self):
+    def _get_connection(self, hostname, username, password):
         client = paramiko.SSHClient()
         client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
-        client.connect(hostname=self.options.host, username=self.options.user,
-                       password=self.options.password)
+        client.connect(hostname=hostname, username=username,
+                       password=password)
         return client
 
-    def _execute_command(self, command):
-        (stdin, stdout, stderr) = self.connection.exec_command(command)
+    def _execute_command(self, connection, command):
+        (stdin, stdout, stderr) = connection.exec_command(command)
         return (stdout, stderr)
 
+    def troubleshoot_mon(self):
+        command = 'sudo ceph health'
```

Comment:
Hi, I might be wrong here, but this is a command going to be run on remote system and hence if the ceph daemon requires sudo on the remote system we need to run sudo command over ssh. One way to get around this that I can think of is maybe have a fallback system whereby the script first tries without sudo and then in an event of failure go with sudo. Please correct me if I am wrong here.